docs/source/statistics.rst

```diff
@@ -27,19 +27,56 @@ Here is a drill-down bar chart summarizing the VerticaPy statistics. Each bar re
 Table: VerticaPy Statistics Summary
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. ipython:: python
-  :suppress:
-
-  # Import the function.
-  from verticapy._utils._inspect_statistics import gen_rst_summary_table
-
-  # Example.
-  print(gen_rst_summary_table())
-  file_path = "statistics.rst"
-  with open(file_path, "w") as rst_file:
-    rst_file.write(gen_rst_summary_table())
+.. .. ipython:: python
+..   :suppress:
+
+..   # Import the function.
+..   from verticapy._utils._inspect_statistics import gen_rst_summary_table
+
+..   # Example.
+..   print(gen_rst_summary_table())
+..   file_path = "statistics.rst"
+..   with open(file_path, "w") as rst_file:
+..     rst_file.write(gen_rst_summary_table())
+
+.. .. include:: ../statistics.rst
+
+
++----------------------------------------+-----------------------+--------------+
```

Comment:
is it automatically generated or you just copy pasted?

Reply:
no it is manual for now.